import numpy as np

# Technically this works, but probably shouldn't. See
#
# https://github.com/numpy/numpy/issues/16366
#
np.maximum_sctype(1)  # E: No overload variant

np.issubsctype(1, np.int64)  # E: incompatible type

np.issubdtype(1, np.int64)  # E: incompatible type
<<<<<<< HEAD
=======

np.find_common_type(np.int64, np.int64)  # E: incompatible type
>>>>>>> c03913b4420b10b4657dfb6a856eb245e1ebfb58
